modules/rabbitmq/src/main/java/org/testcontainers/containers/ContainerConfigurationException.java

```diff
@@ -0,0 +1,7 @@
+package org.testcontainers.containers;
+
+public class ContainerConfigurationException extends RuntimeException {
```

Comment:
Do we really need this custom exception class? I don't mind, just, in general, we use the built-in ones unless there is a need

Reply:
Understood. What would you suggest?

Reply:
Once I got rid of the unused plugins code, this wasn't needed anymore. So I've removed the exception.